src/main/java/mate/academy/service/ConnectionUtil.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.service;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/book_dao_db";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        DB_PROPERTIES.put("user", "root");
+        DB_PROPERTIES.put("password", "password");
```

Comment:
Storing database credentials directly in the source code is a security risk. Consider externalizing these properties to a configuration file or environment variables to enhance security.